.github/workflows/beta-build.yml

```diff
@@ -44,13 +44,12 @@ jobs:
         env:
           NODE_AUTH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
-      # Git config
-      - run: git config --global user.email "beltran.caliz@washpost.com" && git config --global user.name "beltrancaliz"
-        env:
-          NODE_AUTH_TOKEN: ${{ secrets.NPM_PERSONAL_AUTH_TOKEN }}
-
       # Prerelease (tests and lint)
-      - run: npm version prerelease --preid=beta
+      - name: Iterate version beta
```

Comment:
This is good! Assuming the "name" additions will be more helpfully descriptive for each action.